src/main/java/com/redhat/lightblue/camel/request/DataBulkRequestAggregationStrategy.java

```diff
@@ -0,0 +1,33 @@
+package com.redhat.lightblue.camel.request;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.impl.DefaultExchange;
+import org.apache.camel.processor.aggregate.AggregationStrategy;
+
+import com.redhat.lightblue.client.request.AbstractLightblueDataRequest;
+import com.redhat.lightblue.client.request.DataBulkRequest;
+
+/**
+ * Aggregates a collection of {@link AbstractLightblueDataRequest}s into a single {@link DataBulkRequest}.
+ *
+ * @author dcrissman
+ */
+public class DataBulkRequestAggregationStrategy implements AggregationStrategy {
+
+    @Override
+    public Exchange aggregate(Exchange oldExchange, Exchange newExchange) {
```

Comment:
You take request from newExchange and add it to oldExchange - shouldn't the bulk one be new?